.release-please-manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packages/microsoft_kiota_abstractions": "0.0.2",
```

Comment:
I think we need to set the current version here, so 0.0.1 for now

Reply:
Oh you're right. I misread the docs thinking it should contain the next version (for some reason...)